scripts/decrypt-ssl-key.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -e
+FILE_ENC=/vault/config/ssl_private_key.enc
+FILE_CLEAR=/vault/config/ssl_private_key
+echo "On your PC, run"
+echo
+echo "  git clone https://github.com/vimc/montagu-vault"
+echo "  ./montagu-vault/ssl-key/decrypt-key.sh <your-key-name>"
+echo
+echo "where <your-key-name> corresponds to an entry in ssl-keys/pubkeys/"
```

Comment:
Path is wrong, should be `ssl-key/pubkey/` not `ssl-keys/pubkeys`

Reply:
Thanks - updated now